templates/taxi/driver_confirm_delete.html

```diff
@@ -0,0 +1,17 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h1>Delete driver {{ driver.username }}?</h1>
+  <div class="row">
+    <div class="col-1">
+      <a href="{% url 'taxi:driver-detail' driver.id %}"><input type="submit" value="Cancel" class="btn btn-secondary"></a>
```

Comment:
The 'Cancel' button is implemented as a link wrapped around a submit button, which is not semantically correct. Consider using a regular anchor tag styled as a button for the 'Cancel' action, like this: `<a href="{% url 'taxi:driver-detail' driver.id %}" class="btn btn-secondary">Cancel</a>`.

Reply:
changed